Filters/FilterType.cs

```diff
@@ -23,6 +23,7 @@ public enum FilterType
         /// The command is a slider or input field
         /// </summary>
         RANGE = 32,
-        TEXT = 64
+        TEXT = 64,
+        EQUAL_NO_SEARCH = 128
```

Comment:
Could the new type be more generic?
It could be used to disable rich ui features of other filters as well.

Reply:
Like SIMPLE ?

Reply:
Makes sense to me